homeassistant/components/binary_sensor/rfxtrx.py

```diff
@@ -154,12 +153,12 @@ def __init__(self, event, name, device_class=None,
         self._device_class = device_class
         self._off_delay = off_delay
         self._state = False
-        self.is_lighting4 = False
+        self.is_lighting4 = (event.device.packettype == 0x13)
         self.delay_listener = None
         self._data_bits = data_bits
         self._cmd_on = cmd_on
         self._cmd_off = cmd_off
-
+        self.hass = None
```

Comment:
This is done in the Entity class
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/helpers/entity.py#L69

Reply:
My bad, I ran pylint on this file only. Fixed.